scripts/workset_upload.py

```diff
@@ -150,14 +152,11 @@ def __init__(self, lims, crawler, log):
                                 if 'Size (bp)' in inp.udf:
                                     onelib['size']=round(inp.udf['Size (bp)'],2)
 
-                                onelib['concentration']
```

Comment:
yep, it doesnt make much sense.

On Fri, Jul 10, 2015 at 9:10 AM, senthil10 <notifications@github.com> wrote:

> In scripts/workset_upload.py
> <https://github.com/SciLifeLab/LIMS2DB/pull/16#discussion_r34332015>:
>
> > @@ -150,14 +152,11 @@ def __init__(self, lims, crawler, log):
> >                                  if 'Size (bp)' in inp.udf:
> >                                      onelib['size']=round(inp.udf['Size (bp)'],2)
> >
> > -                                onelib['concentration']
>
> @Galithil <https://github.com/Galithil> : I believe this one meant to be
> removed [image: :sweat_smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciLifeLab/LIMS2DB/pull/16/files#r34332015>.
>